editor/src/core/shared/element-template.ts

```diff
@@ -854,7 +854,10 @@ export function jsxAttributesPartReferencesElsewhere(attrPart: JSXAttributesPart
 export function elementReferencesElsewhere(element: JSXElementChild): boolean {
   switch (element.type) {
     case 'JSX_ELEMENT':
-      return element.props.some(jsxAttributesPartReferencesElsewhere)
+      return (
+        element.props.some(jsxAttributesPartReferencesElsewhere) ||
+        element.children.some(elementReferencesElsewhere)
```

Comment:
is this recursive? does it work for children deeply nested inside the element-to-copy?

Reply:
It's recursive (since `elementReferencesElsewhere` checks JSXElements, and through the `children` prop, the all child elements/expressions are checked)